modules/ocf_www/files/build-vhosts

```diff
@@ -53,6 +55,20 @@ class SSL(namedtuple('SSL', ('fqdn',))):
         return str(SSL_BUNDLE / (self.fqdn + '.bundle'))
 
 
+class System_SSL(namedtuple('System_SSL', ('fqdn',))):
```

Comment:
Why the funny naming convention?

Reply:
Dunno, have any better names? I'd be open to suggestions

Reply:
What about `SystemSSL`?

Reply:
Oh, I see what you mean about the naming convention now, that makes more sense. Sure, this would be better